src/FixFasta.cpp

```diff
@@ -485,12 +485,119 @@ spp::sparse_hash_set<std::string> populateDecoyHashPuff(const std::string& fname
   return dset;
 }
 
+bool extractFasta(std::string& inputTsv, std::string& outFile, uint32_t& numFeats) {
```

Comment:
I think this makes sense.  We should think a bit harder about how to reduce potential code duplication.  However, for the time being, I think this should be OK.